gcloud-java-logging/baseline/src/main/java/com/google/gcloud/logging/spi/v2/ConfigServiceV2Api.java

```diff
@@ -53,105 +53,13 @@
 
 // AUTO-GENERATED DOCUMENTATION AND SERVICE - see instructions at the top of the file for editing.
 /**
- * Service Description: See src/api/google/logging/v2/logging.proto for documentation
+ * Service Description:
```

Comment:
"Service Description:" does not look like a nice class level javadoc.
We should ask the service owners to provide description in the service level (or having the generators meta-information include that).

Also, we should add a `@see` entry to https://github.com/googleapis/googleapis/blob/master/google/logging/v2/logging_config.proto